conanfile.py

```diff
@@ -28,7 +28,7 @@ class Clio(ConanFile):
         'protobuf/3.21.9',
         'grpc/1.50.1',
         'openssl/1.1.1u',
-        'xrpl/2.4.0-b1',
+        'xrpl/2.4.0-b1@clio/permission_domains',
```

Comment:
does this need to change?

Reply:
This PR will not be merged until it's in the next beta or something